package.json

```
@@ -5,11 +5,12 @@
   "type": "module",
   "scripts": {
     "build": "vite build",
+    "test": "vitest run --coverage",
```

```suggestion
    "test": "vitest run --coverage  --reporter=verbose --silent",
```
The `--reporter=verbose` I leave it up to you. The `--silent` avoid printing:
```
stderr | issueWarning (file:/home/rts/dev/pillarbox-web-theme-editor/node_modules/@lit/reactive-element/src/reactive-element.ts:92:15)
Lit is in dev mode. Not recommended for production! See https://lit.dev/msg/dev-mode for more information.
```

Maybe there is a better way to disable this console.log.

---

I've opted to add both as suggested.